index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="keywords" content="">
+    <title>Attr🔹ct Quiz</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="index.css" rel="stylesheet">
+    <link rel="icon" href="images/cover.png">
+    <link href="https://attr-act.netlify.app/Attract.css" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <div class="header-left">
+            
+            <button class="secondary-link"><a href="index.html">Home</a></button>
+        </div>
+    <h1 class="headings">Attr🔹ct Quiz</h1>
+    <div class="header-right">
+        <a href="https://github.com/GuptaEesh" target="_blank" rel="noopener" rel="noreferrer"><img
+                class="img-xsm profile-avatar" src="https://cdn-icons-png.flaticon.com/512/25/25231.png"
+                alt="Github icon"></a>
+        <a href="https://twitter.com/ra1711003010629" target="_blank" rel="noopener" rel="noreferrer"><img
+                class="img-xsm profile-avatar" src="https://cdn-icons-png.flaticon.com/512/1051/1051382.png"
+                alt="Twitter icon"></a>
+        <a href="https://webdevjourney.hashnode.dev/" target="_blank" rel="noopener" rel="noreferrer"><img
+                class="img-xsm profile-avatar"
+                src="https://cdn.hashnode.com/res/hashnode/image/upload/v1611902473383/CDyAuTy75.png?auto=compress"
+                alt="hashnode icon"> </a>
+        <a href="https://www.linkedin.com/in/eesh-gupta-42673a144" target="_blank" rel="noopener"
+            rel="noreferrer"><img class="img-xsm profile-avatar"
+                src="https://cdn-icons-png.flaticon.com/512/174/174857.png" alt="Linkedin icon"> </a>
+    </div>
+</header>
+<main class="quiz-body">
+    
+    <div class="quiz-options">
+    <a href="quizes/designQuiz.html"><div class="eg-card card-two">
+        <header class="card-header-text">Design Quiz</header>
+        <img class="card-image" src="images/design.jpg" alt="car design">
+        <main class="card-body">
+              <p>When it does not exist, design it.<br> <b>"Henry Royce"</b></p>
+        </main>
+        <footer class="card-footer-icons">
+                    <button class="tertiary md">Read more about car design</button>
+        </footer>
```

Comment:
Not the best way to use semantic tags, they should ideally follow the below order and convention

```html
<header></header>
<main></main>
<footer></footer>
```

Or instead of using those tags, you can use ``` <section>``` tag to split your content in each card

Reply:
Done✅